modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -755,12 +755,12 @@ export async function verifyWalletSignature(params: {
   const publicUValueRawNotationIndex = 2 + params.verifierIndex;
 
   // Derive public form of u-value
-  const publicUValue = ecc.pointFromScalar(Buffer.from(params.decryptedShare.slice(0, 64), 'hex'), false);
+  const publicUValue = ecc.pointFromScalar(Buffer.from(params.decryptedShare.slice(0, 64), 'hex'), true);
+  assert(publicUValue !== null, 'invalid value');
```

Comment:
i would make this a little more descriptive

Reply:
I wasn't sure how far to go for error messages regarding leaking information or anything like that